src/__tests__/snapshot-tests/documenter.test.ts

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 import { getAllComponents, requireComponentDefinition } from '../utils';
 
-describe('Documenter', () => {
+describe.skip('Documenter', () => {
```

Comment:
If you are wondering why no diff in the `__snapshots__` file, snapshots for skipped tests are still here. This will help me to re-enable them and render the diff only for the changed ones